### PR TITLE
Cardano RPC documentation

### PR DESCRIPTION
**Pull request by Hactive808**

### CODE REVIEW

File: docs/rpc/cardano-rpc-documentation/constructionPayloads.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  - `amount` (object, required): An object containing information about the transaction amount.
+    - `value` (string, required): The value of the transaction amount.
+    - `currency` (object, required): An object specifying the currency details.
+      - `symbol` (string, required): The symbol or code of the currency.
+      - `decimals` (number, required): The number of decimal places for the currency.
+      - `metadata` (object, optional): Any additional information related to the currency itself. For example, it would be useful to populate this object with the contract address of an ERC-20 token.
+    - `metadata` (object, optional): metadata object for the amount.
+  - `coin_change` (object, required): An object containing information about coin changes in the operation.
+    - `coin_identifier` (object, required): An object containing a coin identifier.
+      - `identifier` (string, required): Identifier should be populated with a globally unique identifier of a Coin. In Bitcoin, this identifier would be transaction_hash:index..
```
Comment (Hathoriel) [inline docs/rpc/cardano-rpc-documentation/constructionPayloads.md:138]: Bitcoin?

File: docs/rpc/cardano-rpc-documentation/constructionPreprocess.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  - `amount` (object, required): An object containing information about the transaction amount.
+    - `value` (string, required): The value of the transaction amount.
+    - `currency` (object, required): An object specifying the currency details.
+      - `symbol` (string, required): The symbol or code of the currency.
+      - `decimals` (number, required): The number of decimal places for the currency.
+      - `metadata` (object, optional): Any additional information related to the currency itself. For example, it would be useful to populate this object with the contract address of an ERC-20 token.
+    - `metadata` (object, optional): metadata object for the amount.
+  - `coin_change` (object, required): An object containing information about coin changes in the operation.
+    - `coin_identifier` (object, required): An object containing a coin identifier.
+      - `identifier` (string, required): Identifier should be populated with a globally unique identifier of a Coin. In Bitcoin, this identifier would be transaction_hash:index..
```
Comment (Hathoriel) [inline docs/rpc/cardano-rpc-documentation/constructionPreprocess.md:132]: Bitcoin?

Comment (Hactive808) [reply docs/rpc/cardano-rpc-documentation/constructionPreprocess.md:132]: This is just example and its even mentioned by official Cardano docs

File: docs/rpc/cardano-rpc-documentation/constructionPayloads.md (hunk 1 above)
Comment (Hactive808) [reply docs/rpc/cardano-rpc-documentation/constructionPayloads.md:138]: This is just example and its even mentioned by official Cardano docs

**Review by Hathoriel — APPROVED** [review]

**Pull request merged**